### ISSUE TITLE
renovation - PRJ_v1.0

### ISSUE THREAD

**Issue by ran-codes** [Drexel-UHC/salurbal-fair-renovations#57]
## Action Items 
  - [x] ETL setup
  - [x] define data-metadata relationships -> **linkage.csv**
  - [x] generate codebook templates -> **4.0-template__codebooks.xlsx**
  - [x] initial filling out of codebooks  -> **4.1-raw__codebooks.xlsx**
  - [x] QC of initial codebooks -> **4.2-base__codebooks.xlsx**
  - [ ] Review of base codebooks  -> **4.3-int__codebooks.xlsx**
  - [ ] QC of intermediate codebooks  -> **4.4-final__codebooks.xlsx**
  - [ ] Integration and loading into data warehouse

**Comment by juruchi** [comment]
@goro-yamada You mentioned that although none of the single age population projections will be made public, some of the aggregated data will be publicly available. Will the data in the aggregated datasets (PRJ, PRJG, PRJT) be made public in the same way? i.e. If a country's PRJ data is made available, PRJG and PRJT data will also be available.

**Comment by juruchi** [comment]
@goro-yamada Per our zoom conversation, all countries are expected to be made public in the same way across the aggregated datasets except in the case of AR. Conversations are ongoing with the country team.

**Comment by goro-yamada** [comment]
@juruchi That's right. Other countries have consistency between PRJ, PRJG, and PRJT. AR, CR, and PA's data are not open to the public as of now.

**Comment by ran-codes** [comment]
### Summary

- PRJS: is the parent of everything. It will use to compile 'source' for all downstream datasets. but PRJS is not to be made public
- downstream dataset (PRJ, PJG, PJT)
   - source will come from PRJS
   - All countries but AR, CR, PA: the public status only needs to fill in one place. 
   - AR, CR, PA: the public status will differ by dataset (e.g betwe PRJ, PRJG, PRJT).

### Example template for Goro

This sheet will control public for all PRJ's
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ranli/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ranli/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



iso2 | dataset | public
-- | -- | --
PE | PRJ; PRJG; PRJT | 1
CL | PRJ; PRJG; PRJT | 0
MX | PRJ; PRJG; PRJT | 1
AR | PRJ | 1
AR | PRJG | 0
AR | PRJT | 1
CR | PRJ | 0
CR | PRJG | 0
CR | PRJT | 1



</body>

</html>

### Action Items

- [ ] @juruchi to operationalzie full combination as a .xlsx or .csv and send to goro to fill out.
- [ ] @ran-codes build logic to compile public for all PRJ datasets using the Sheet that goro fills out

**Comment by goro-yamada** [comment]
@ran-codes @juruchi 
- All countries but AR, CR, PA: the public status only needs to fill in one place.
- AR, CR, PA: the public status will differ by dataset (e.g betwe PRJ, PRJG, PRJT)._
No. It's only AR, not CR and PA.

**Comment by usamabilal** [comment]
Goro, isn’t the parent PRJ? PRJS is derived right ?
________________________________

**Comment by goro-yamada** [comment]
That's true, Usama, in most cases. One (or several) countries provide single-year data, although I think statistics; offices estimate single-year age from five-year age data or with more complex formulas.

I kept quiet about which is considered the "parent" because the final results will be the same. But, yes, 5-year age data are the base for other datasets in data processing.

**Comment by juruchi** [comment]
Attached is a document that centralizes 'source' and 'public' info for all the PRJ datasets. For public, I put together what I believe are all the combinations we have at the moment based on:
- No PRJS will be public
- All countries except for AR will have public synced for PRJ, PRJG, PRJT
- AR may have different public values for PRJ, PRJG, PRJT

@goro-yamada Can you review the file and make corrections as needed?
[source_public_prj_datasets.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/files/11501643/source_public_prj_datasets.xlsx)

**Comment by goro-yamada** [comment]
@juruchi Done! I changed the public code for CR and PA from 0 to 9.
[source_public_prj_datasets.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/files/11503645/source_public_prj_datasets.xlsx)

**Comment by juruchi** [comment]
Thanks!

@ran-codes the raw codebooks for PRJ and PRJG #20 are ready except for these variables and 'acknowledgement' which would be the same as 'source'.

**Comment by ran-codes** [comment]
@goro-yamada The codebook is ready for your review. Details on codebook fields can be found [here](https://github.com/Drexel-UHC/salurbal-fair-renovations/raw/main/documents/templates/_template_details.xlsx) Let us know if you have any questions!

## Action items

- [ ] Review **UPDATED CODEBOOK**  [4.2-base__codebook.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/files/11554010/4.2-base__codebook.xlsx)
- [ ] Review [associated sources for {{population}} ](https://github.com/Drexel-UHC/salurbal-fair-renovations/raw/main/datasets/_src/src__population.xlsx)
- [ ] Make edits as needed and save in a file called **4.3-int__codebook.xlsx**
- [ ] Comment in this issue when the final version is ready and upload the file to this issue via drag and drop

**Comment by goro-yamada** [comment]
I have the same comments as for #20. Source institutions were with or without accents. I corrected them in the codebook, but I DID NOT update the source for the population (it seemed the same as I saw in #20).
I also updated the first age category. It is for age 0, not for age 0-1,
[4.3-int__codebook.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/files/11605813/4.3-int__codebook.xlsx)
